tests/test_subset_wfs_polygon.py

```diff
@@ -0,0 +1,39 @@
+from pywps import Service
+from pywps.tests import client_for, assert_response_success
+
+from .common import get_output
+from flyingpigeon.processes.wps_subset_wfs_polygon import SubsetWFSPolygonProcess
+
+
+def test_wps_subset():
+    client = client_for(Service(processes=[SubsetWFSPolygonProcess()]))
+
+    datainputs = "resource=@xlink:href={nc};" \
+                 "typename={typename};" \
+                 "geoserver={geoserver};" \
+                 "featureids={fid1};" \
```

Comment:
The WPS requests expects individual entries per featureid?

Reply:
Not sure I understand the question. You may send multiple featureids. Birdy converts a list or tuple into those individual entries in the HTML request.

Reply:
Any idea how to tell codacy this is python 3 only ?

Reply:
I've configured the *codacy patterns* to use `PyLint (Python 3)`  ...  but it didn't help.